lib/browser.js

```diff
@@ -1,9 +1,24 @@
+import { log } from './logger'
```

Comment:
we should probably decide if we go for require, as in Line 3 or with esm imports. Let's go esm?

Reply:
sounds good

Reply:
@cilice, now it is esm